### PR TITLE
chore(sql): tweak initial set sizes in count_distinct functions

### PR DESCRIPTION
**Pull request by puzpuzpuz**
Size of 64 led to OOMEs in one of ClickBench queries, so this patch makes sure that we use 16 and 0.7 as the initial size and load factor for all hash sets used in `count_distinct()`.

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 8 / 8 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctUuidGroupByFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctStringGroupByFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/std/Long256HashSet.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctIPv4GroupByFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctLongGroupByFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctLong256GroupByFunction.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/std/CompactCharSequenceHashSet.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctIntGroupByFunction.java|1|1|100.00%|

**Review by jerrinot — DISMISSED** [review]

**Comment by puzpuzpuz** [comment]
@jerrinot thanks for the quick review!

File: core/src/main/java/io/questdb/griffin/engine/functions/groupby/CountDistinctIPv4GroupByFunction.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public void clear() {
     public void computeFirst(MapValue mapValue, Record record) {
         final CompactIntHashSet set;
         if (sets.size() <= setIndex) {
-            sets.extendAndSet(setIndex, set = new CompactIntHashSet(64, 0.7, Numbers.IPv4_NULL));
+            sets.extendAndSet(setIndex, set = new CompactIntHashSet(16, 0.7, Numbers.IPv4_NULL));
```
Comment (bluestreak01) [inline core/src/main/java/io/questdb/griffin/engine/functions/groupby/CountDistinctIPv4GroupByFunction.java:59]: @puzpuzpuz could we have this  pulled out as a configuration please?

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: core/src/main/java/io/questdb/griffin/engine/functions/groupby/CountDistinctIPv4GroupByFunction.java (hunk 1 above)
Comment (puzpuzpuz) [reply core/src/main/java/io/questdb/griffin/engine/functions/groupby/CountDistinctIPv4GroupByFunction.java]: Good idea. Done in 4e2cb4e075ffc8e7a7d58f67f926f00a25359181

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 62 / 65 (95.38%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/CairoConfigurationWrapper.java|0|3|00.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctUuidGroupByFunction.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctIPv4GroupByFunctionFactory.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctSymbolGroupByFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctIntGroupByFunctionFactory.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctLong256GroupByFunctionFactory.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctLongGroupByFunctionFactory.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctStringGroupByFunction.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctStringGroupByFunctionFactory.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cairo/DefaultCairoConfiguration.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctSymbolGroupByFunction.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/PropertyKey.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/std/Long256HashSet.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctIPv4GroupByFunction.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctLongGroupByFunction.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctLong256GroupByFunction.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/std/CompactCharSequenceHashSet.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctUuidGroupByFunctionFactory.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/groupby/CountDistinctIntGroupByFunction.java|4|4|100.00%|

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**